tasks/debian.yml

```diff
@@ -0,0 +1,40 @@
+- name: "Install exim (exim4) the mail transfer agent"
+  apt: >
+    name=exim4-daemon-heavy
+    state=present
+  tags:
+    - exim
+
+- name: "Make sure exim (exim4) is running"
+  service: >
+    name=exim4
+    state=started
+    enabled=yes
+  tags:
+    - exim
+
+
+- name: "Configure the package"
+  shell: |
+    executable=/bin/bash
+    dpkg-reconfigure -f noninteractive exim4-config &&\
+    update-exim4.conf
+  tags:
+    - exim
+
+- name: "Copy the template configuration"
+  template: >
+    src=update-exim4.conf
+    dest=/etc/exim4/update-exim4.conf
+    owner=root
+    group=root
+    mode=0644
+  tags:
+    - exim
+
+- name: "Restart the exim (exim4) service"
+  service: >
+    name=exim4
+    state=restarted
```

Comment:
That and move the restart to a [handler](https://docs.ansible.com/playbooks_intro.html#handlers-running-operations-on-change) and add a notify to the config task